examples/common/apps.ts

```diff
@@ -0,0 +1,44 @@
+// List of all e2e tested apps
+const apps = [
+  // examples
+  "api",
+  "create-next-app",
+  "middleware",
+  "vercel-blog-starter",
+  "vercel-commerce",
+  // // e2e
+  "app-pages-router",
+  "app-router",
+  "pages-router",
+  // // bugs
+  "gh-119",
+  "gh-219",
+  "gh-223",
+] as const;
```

Comment:
I don't really like this listing and I don't think it is necessary

I think things could be simplified here by either:
 - (worse) keeping the indexes in the global scope and increment them in `getAppPort` and `getInspectorPort`
 - (better) implementing a generic "getNextAvailablePort(start: number)" that both `getAppPort` and `getInspectorPort` use
 
 I think that with either of the above the solution here would be made more generic and easier to maintain (the `getNextAvailablePort` would also make this more robust)
 
 Anyways this is already an improvement to what we currently have, so if my comment is not blocking

Reply:
> implementing a generic "getNextAvailablePort(start: number)" that both getAppPort and getInspectorPort use

There is no common state across apps, so we would need to sync using i.e. a file. Not sure if worth the burden.